pkg/relayer/config/types.go

```diff
@@ -66,13 +70,14 @@ type YAMLRelayMinerMetricsConfig struct {
 // YAMLRelayMinerSupplierConfig is the structure used to unmarshal the supplier
 // section of the RelayMiner config file
 type YAMLRelayMinerSupplierConfig struct {
-	ListenUrl             string                              `yaml:"listen_url"`
-	ServiceConfig         YAMLRelayMinerSupplierServiceConfig `yaml:"service_config"`
-	ServiceId             string                              `yaml:"service_id"`
-	SigningKeyNames       []string                            `yaml:"signing_key_names"`
-	RequestTimeoutSeconds uint64                              `yaml:"request_timeout_seconds"`
-	MaxBodySize           string                              `yaml:"max_body_size"`
-	XForwardedHostLookup  bool                                `yaml:"x_forwarded_host_lookup"`
+	ListenUrl             string                                         `yaml:"listen_url"`
+	ServiceConfig         YAMLRelayMinerSupplierServiceConfig            `yaml:"service_config"`
```

Comment:
Please update the docs explaining this will be the default config.

Add a TODO_TECHDEBT (in the code AND in the docs as an admonition) to rename it to DefaultService Config.